crates/bevy_text/src/text.rs

```diff
@@ -170,11 +170,13 @@ impl TextLayout {
 /// # use bevy_ecs::world::World;
 /// # use bevy_text::{Font, TextLayout, TextFont, TextSpan, TextColor};
 /// # use bevy_hierarchy::BuildChildren;
-///
+/// #
 /// # let font_handle: Handle<Font> = Default::default();
 /// # let mut world = World::default();
 /// #
 /// world.spawn((
+///     // `Text` or `Text2d` are needed, and will provide default instances
+///     // of the following components.
 ///     TextLayout::default(),
```

Comment:
Shouldn't this have a `Text` or `Text2d` component here as part of the example?

Reply:
Yep, but this is in `bevy_text`, which doesn't have `bevy_ui` and/or `bevy_sprite` as a dependency, so I don't think we can do that?

Reply:
Yeah I think that would be best.